PC/winreg.c

```diff
@@ -270,7 +297,11 @@ static PyObject *
 winreg_HKEYType_Close_impl(PyHKEYObject *self)
 /*[clinic end generated code: output=fced3a624fb0c344 input=6786ac75f6b89de6]*/
 {
-    if (!PyHKEY_Close((PyObject *)self))
+    PyObject *m = PyType_GetModule(Py_TYPE(self));
+    if (m == NULL) {
+        return NULL;
+    }
```

Comment:
I'd make this an assert.

```suggestion
    assert(m != NULL);
```